docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -590,11 +608,11 @@ docker service create \
     taskmanager
 ```
 
-The *job artifacts* must be available in the *JobManager* container, as outlined [here](#start-a-job-cluster).
+The *job artifacts* must be available in the JobManager container, as outlined [here](#application-mode-on-docker).
 See also [how to specify the JobManager arguments](#jobmanager-additional-command-line-arguments) to pass them
 to the `flink-jobmanager` container.
 
 The example assumes that you run the swarm locally and expects the *job artifacts* to be in `/host/path/to/job/artifacts`.
-It also mounts the host path with the artifacts as a volume to the container's path `/opt/flink/usrlib`.
+It also mounts the host path with the artifacts as a volume to the container's path `/opt/flink/usrlib`.    
```

Comment:
```suggestion
It also mounts the host path with the artifacts as a volume to the container's path `/opt/flink/usrlib`.
```

Reply:
Sorry for this stuff. I should check the diff myself.